tests/base_test_case.py

```diff
@@ -135,6 +135,7 @@ def assertLazyMessage(self, msg_func, assert_function, *args, **kwargs):
     # equivalent to python 2.7's unittest.assertRegexpMatches()
     def assertRegexpMatches(
             self, text, expected_regexp, msg="Regexp didn't match"):
+        msg = '%s: %r not found in %r' % (msg, expected_regexp, text)
```

Comment:
This is problematic for multi-line stuff. BaseMainCase._format_expected_actual might be borrow/adapt-able to this.